exchange/bidder_test.go

```diff
@@ -1401,6 +1422,23 @@ func TestTimeoutNotificationOn(t *testing.T) {
 	assert.EqualValues(t, logExpected, logActual)
 }
 
+func TestParseDebugInfoTrue(t *testing.T) {
+	debugInfo := &config.DebugInfo{Allow: true}
+	resDebugInfo := parseDebugInfo(debugInfo)
+	assert.True(t, resDebugInfo.Allow, "Debug Allow value should be false")
+}
+
+func TestParseDebugInfoFalse(t *testing.T) {
+	debugInfo := &config.DebugInfo{Allow: false}
+	resDebugInfo := parseDebugInfo(debugInfo)
+	assert.False(t, resDebugInfo.Allow, "Debug Allow value should be false")
+}
+
+func TestParseDebugInfoIsNil(t *testing.T) {
+	resDebugInfo := parseDebugInfo(nil)
+	assert.True(t, resDebugInfo.Allow, "Debug Allow value should be false")
```

Comment:
Same as above, just need to change failure message to true

Reply:
Fixed